tests/integration/parity/common.py

```diff
@@ -97,6 +97,10 @@ def test_eth_getTransactionReceipt_unmined(self, web3, unlocked_account):
         assert receipt is not None
         assert receipt['blockHash'] is None
 
+    def test_eth_getLogs_with_logs_none_topic_args(self, web3):
+        with pytest.raises(AssertionError):
```

Comment:
In all the other places I can think of, we do `@pytest.mark.xfail` instead of this approach. Nothing wrong with this that I can think of, it's just nice to have a little more stylistic consistency.